src/current/cockroachcloud/production-checklist.md

```diff
@@ -66,6 +66,20 @@ When several transactions try to modify the same underlying data concurrently, t
 
 To ensure optimal SQL performance for your CockroachDB {{ site.data.products.cloud }} cluster, follow the best practices described in the [SQL Performance Best Practices](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/performance-best-practices-overview) guide.
 
+## Configure node shutdown settings
+
+Clusters in CockroachDB {{ site.data.products.cloud }} have the following default values for [cluster settings]({% link {{ site.current_cloud_version }}/cluster-settings.md %}) that relate to [node shutdown]({% link {{ site.current_cloud_version }}/node-shutdown.md %}) for maintenance, upgrades, or scaling. Depending on the requirements of your applications and workloads, you may need to modify `server.shutdown.connections.timeout` or `server.shutdown.transactions.timeout`.
+
+Cluster setting | Default | Details
+----------------|---------|---------
+[`server.shutdown.initial_wait`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-drain-wait)<br />Alias: `server.shutdown.drain_wait` | 3 minutes | How long to wait before redirecting new client requests to non-draining nodes. After this duration has elapsed, the node no longer accepts new client connections. This setting cannot be changed for clusters in CockroachDB {{ site.data.products.cloud }}.
+[`server.shutdown.connections.timeout`]({% link {{ site.current_cloud_version }}/cluster-settings.md %}#setting-server-shutdown-connection-wait)<br />Alias: `server.shutdown.connection_wait` | 0 seconds | How long after `server.shutdown.initial_wait` completes to wait for client connections to drain before forcibly disconnecting them from the node. Can be set to a value between 0 and 30 minutes (1800 seconds).
```

Comment:
I'd reword this a bit: "how long a crdb node waits for client SQL connections to close before forcibly closing the connection". 

Below, I'd also note the tradeoff involved when setting this value. Increasing this value will result in slower cluster operations (version upgrades, decommissioning nodes, etc). Each node shutdown will now wait `server.shutdown.connections.timeout` as part of the shutdown process. But increasing it may be required to avoid connection interruptions during these maintenance ops.

Reply:
Done